pytorch_tabnet/utils.py

```diff
@@ -58,24 +58,32 @@ def create_dataloaders(
 
     Parameters
     ----------
-        X_train: np.ndarray
-            Training data
-        y_train: np.array
-            Mapped Training targets
-        X_valid: np.ndarray
-            Validation data
-        y_valid: np.array
-            Mapped Validation targets
-        weights : either 0, 1, dict or iterable
-            if 0 (default) : no weights will be applied
-            if 1 : classification only, will balanced class with inverse frequency
-            if dict : keys are corresponding class values are sample weights
-            if iterable : list or np array must be of length equal to nb elements
-                          in the training set
+    X_train : np.ndarray
+        Training data
+    y_train : np.array
+        Mapped Training targets
+    eval_set : list of tuple
+        List of eval tuple set (X, y)
+    weights : either 0, 1, dict or iterable
+        if 0 (default) : no weights will be applied
+        if 1 : classification only, will balanced class with inverse frequency
+        if dict : keys are corresponding class values are sample weights
+        if iterable : list or np array must be of length equal to nb elements
+                      in the training set
+    batch_size : int
+        how many samples per batch to load
+    num_workers : int
+        how many subprocesses to use for data loading. 0 means that the data
+        will be loaded in the main process
+    drop_last : bool
+        set to True to drop the last incomplete batch, if the dataset size is not
+        divisible by the batch size. If False and the size of dataset is not
+        divisible by the batch size, then the last batch will be smaller
+
```

Comment:
Could you add `pin_memory` please?
It's a boolean value that decides whether to pin gpu memory during training.